src-ui.v3/src/Main.hs

```diff
@@ -136,6 +138,27 @@ bodyElement4 = do
       display (V.length <$> dynPackages0)
       text ")"
 
+    -- search box
```

Comment:
This section has some weird differences in indentation if we care. 4 spaces vs 2 spaces by the looks.

Reply:
noted because at the top it is 4 spaces. I'll clean it up in the next commit.